rails-commenting.rb

```diff
@@ -10,21 +10,25 @@
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    #  this @posts variable is containing all the information on the BlogPost Table
     @posts = BlogPost.all
   end
 
   def show
     # ---3)
+    #  this @post variable contains a parameter that allows the end user to call on certain information contain in the BlogPost table
```

Comment:
Yes, now the instance variable is holding the post that has that id passed into it